gulpfile.js

```diff
@@ -34,19 +34,20 @@ var moduleMap = {};
 for (var key in fbjsModuleMap) {
   moduleMap[key] = fbjsModuleMap[key];
 }
-var whiteListNames = [
-  'deepDiffer',
-  'deepFreezeAndThrowOnMutationInDev',
-  'flattenStyle',
-  'InitializeJavaScriptAppEngine',
-  'RCTEventEmitter',
-  'TextInputState',
-  'UIManager',
-  'View',
-];
+var whiteList = {
+  deepDiffer: 'react-native/lib/deepDiffer',
+  deepFreezeAndThrowOnMutationInDev: 'react-native/lib/deepFreezeAndThrowOnMutationInDev',
+  flattenStyle: 'react-native/lib/flattenStyle',
+  InitializeJavaScriptAppEngine: 'react-native/lib/InitializeJavaScriptAppEngine',
+  RCTEventEmitter: 'react-native/lib/RCTEventEmitter',
+  TextInputState: 'react-native/lib/TextInputState',
+  UIManager: 'react-native/lib/UIManager',
+  UIManagerStatTracker: 'react-native/lib/UIManagerStatTracker',
+  View: 'react-native/lib/View',
+};
 
-whiteListNames.forEach(function(name) {
-  moduleMap[name] = name;
+Object.keys(whiteList).forEach(function(name) {
+  moduleMap[name] = whiteList[name];
 });
 
 moduleMap['object-assign'] = 'object-assign';
```

Comment:
sure, I can change that. Having the renderer in its own package makes a lot of sense.